src/Paket.Core/PaketConfigFiles/ProjectFile.fs

```diff
@@ -1180,11 +1180,20 @@ module ProjectFile =
 
 
     let save forceTouch project =
+        let hasUtf8Bom fileName =
```

Comment:
Should this go in another place? I kept it as a local function because it's only used here.